traffic_portal/app/src/common/directives/dragAndDrop/droppableDirective.js

```diff
@@ -0,0 +1,146 @@
+var DroppableDirective = function () {
+    return {
+        scope: {
+            drop: '&',
+            dragsmart: '&',
+            bin: '='
+        },
+        link: function(scope, element) {
+            var el = element[0];
+
+            el.parentElement.classList.add('grabbable');
+
+            el.addEventListener(
+                'mouseover',
+                function(e) {
+                    this.classList.add('hovering');
```

Comment:
Do you need this event listener? Is there a reason you can't write your CSS rule with `.grabable:hover` instead of `.hovering`?

Reply:
done. i moved it into fallback:hover instead